### DISCUSSION TITLE
DomainReloading support?

### DISCUSSION THREAD

**Discussion by Racot007** [KyryloKuzyk/PrimeTween#2] [General] [answered]
Hi there. 

Nice asset. I can see you use Statics. Can he support [DomainReloading](https://docs.unity3d.com/Manual/DomainReloading.html)?

**Comment by KyryloKuzyk** [comment, marked as answer]
Of course! I use the disabled 'Reload Domain' setting all the time and PrimeTween is built with this setting in mind.

PrimeTween even works fine with the 'Recompile And Continue Playing' setting: if you re-compile your project on the fly, PrimeTween will stop all running tweens and will continue to work with no problems.
